cookbook/enums.md

```diff
@@ -0,0 +1,51 @@
+# Enums
+
+Enums belong to entityInfo part of metadata, which means using a particular schema version does not mean enum values won't change. Adding an enum value is considered a non breaking change and should be allowed. Removing or renaming an enum is a breaking change and should not be allowed.
+
+Considering the above, you have 2 options when designing model classes in a language which supports enum types (such as java):
+* don't use enum type, i.e. define enum metadata fields as strings in the model classes or
+* implement enum types which support unknowns.
+
+An example of a java enum with pojo2java nad java2pojo conversion support for the unknowns:
```

Comment:
This supports only one "unknown", not unknowns. This isn't a good suggestion.

a=EnumType.fromValue("z") -> unknown
a.toString() -> z
b=EnumType.fromValue("y") -> unknown
EnumType.unknown.toString() -> y
a.toString() ->y